charts/images.yaml

```diff
@@ -0,0 +1,17 @@
+images:
+- name: kubevirt-cloud-controller-manager
+  sourceRepository: github.com/kubevirt/cloud-provider-kubevirt
+  repository: docker.io/mfranczy/kubevirt-cloud-controller-manager
+  tag: latest
+- name: machine-controller-manager
+  sourceRepository: github.com/gardener/machine-controller-manager
+  repository: eu.gcr.io/gardener-project/gardener/machine-controller-manager
+  tag: "v0.33.0"
+- name: machine-controller-manager-provider-kubevirt
+  sourceRepository: github.com/gardener/machine-controller-manager-provider-kubevirt
+  repository: docker.io/mfranczy/machine-controller-manager-provider-kubevirt
```

Comment:
Could we host this image somewhere under eu.gcr.io/gardener-project/gardener?

Reply:
Yes, it would be the best, I asked about that in the past, however, I don't have enough permissions to do it and there was a suggestion to push it to my private repo just for now. If you could help me with permissions issue or give me instruction about the process how you push images to gcr then that would be great.

Reply:
I don't find this acceptable, even at this point. Let's try to give you permissions, or perhaps I could find a workaround.

Reply:
Please fix this, you should now have permissions to push into our docker registry.

Reply:
done